### PR TITLE
Mage_ImportExport - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- doc blocks added/fixed
- PSR2 fixes (whitespaces, linebreaks, ...)

### CODE REVIEW

File: app/code/core/Mage/ImportExport/Model/Import/Uploader.php [hunk 1]
```diff
@@ -43,6 +43,10 @@ class Mage_ImportExport_Model_Import_Uploader extends Mage_Core_Model_File_Uploa
     );
     const DEFAULT_FILE_TYPE = 'application/octet-stream';
 
+    /**
+     * Mage_ImportExport_Model_Import_Uploader constructor.
+     * @param null $filePath
```
Comment (kkrieger85) [inline app/code/core/Mage/ImportExport/Model/Import/Uploader.php:48]: This should be "string"?

Comment (kkrieger85) [reply app/code/core/Mage/ImportExport/Model/Import/Uploader.php:48]: Or string|null

Comment (tmotyl) [inline app/code/core/Mage/ImportExport/Model/Import/Uploader.php:48]:
```suggestion
     * @param string|null $filePath
```

Comment (tmotyl) [reply app/code/core/Mage/ImportExport/Model/Import/Uploader.php:48]: right

**Pull request merged**